SaintCoinach/Definitions/ContentFinderCondition.json

```
@@ -90,63 +90,63 @@
       }
     },
     {
-      "index": 16,
+      "index": 17,
       "name": "ClassJobLevel{Required}"
     },
     {
-      "index": 17,
+      "index": 18,
       "name": "ClassJobLevel{Sync}"
     },
     {
-      "index": 18,
+      "index": 19,
       "name": "ItemLevel{Required}"
     },
     {
-      "index": 19,
+      "index": 20,
       "name": "ItemLevel{Sync}"
     },
     {
-      "index": 21,
+      "index": 22,
       "name": "AllowUndersized"
     },
     {
-      "index": 23,
+      "index": 24,
       "name": "AllowReplacement"
     },
     {
-      "index": 25,
+      "index": 26,
       "name": "AllowExplorerMode"
     },
     {
-      "index": 31,
+      "index": 32,
       "name": "HighEndDuty"
     },
     {
-      "index": 36,
+      "index": 37,
       "name": "DutyRecorderAllowed"
     },
     {
-      "index": 41,
+      "index": 42,
       "name": "Name"
     },
     {
-      "index": 42,
+      "index": 43,
       "name": "NameShort"
     },
     {
-      "index": 43,
+      "index": 44,
       "name": "ContentType",
       "converter": {
         "type": "link",
         "target": "ContentType"
       }
     },
     {
-      "index": 44,
+      "index": 45,
       "name": "TransientKey"
     },
     {
-      "index": 45,
+      "index": 46,
```

```suggestion
      "index": 47,
```
There was another field added at index 45 that pushed this one over to index 47